Don't force -fstack-protector, the toolchain might lack support for it.

Signed-off-by: Gustavo Zacarias <gustavo@zacarias.com.ar>

diff -Nura radvd-2.1.orig/Makefile.am radvd-2.1/Makefile.am
--- radvd-2.1.orig/Makefile.am	2014-07-22 08:23:12.642260069 -0300
+++ radvd-2.1/Makefile.am	2014-07-22 08:26:42.651380759 -0300
@@ -43,7 +43,6 @@
 
 AM_CFLAGS = \
 	-fno-strict-aliasing \
-	-fstack-protector \
 	$(ENABLE_WARNINGS) \
 	$(DISABLE_WARNINGS)
 
